flink-table/flink-table-planner-blink/src/main/scala/org/apache/flink/table/planner/codegen/CodeGeneratorContext.scala

```diff
@@ -108,6 +108,11 @@ class CodeGeneratorContext(val tableConfig: TableConfig) {
     */
   private var currentMethodNameForLocalVariables = "DEFAULT"
 
+  /**
+   * Flag indicating whether split has occurred.
+   */
+  private var isSplit = false
```

Comment:
Rename the variable to `isCodeSplit` to make it clear that the **code** is splitted? 
The comment can also be improved `Flag that indicates whether the generated code is split into several methods.`